test/bats/public-ln-receive.bats

```diff
@@ -127,3 +233,77 @@ usd_amount=50
   # Check for settled
   retry 15 1 check_ln_payment_settled "$payment_request"
 }
+
+@test "public-ln-receive: fail to create invoice - nonexistent wallet-id" {
```

Comment:
this is not the proper description, this could be `fail to create invoice - invalid wallet id` but you need to create another test with a random UUID to get the `nonexistent wallet-id` error

Reply:
Good catch, fixed https://github.com/GaloyMoney/galoy/pull/2971/commits/0755cf7bbcc9a8861f7431bb85d187879dbbd559